ReactSkia/sdk/CurlNetworking.cpp

```diff
@@ -104,6 +139,16 @@ void CurlNetworking::processNetworkRequest(CURLM *curlMultiHandle) {
 
 }
 
+
+bool CurlNetworking::insertIntoCache(const char* key, folly::dynamic &headerBuffer, char* responseBuffer, CurlNetworking* self){
+  valueFactory cacheData;
```

Comment:
Create structure with constructor with initializer list. And can remove this function itself